tf_encrypted/convert/register.py

```diff
@@ -177,6 +178,43 @@ def _keras_conv2d(converter, interiors, inputs):
 
   return out
 
+def _keras_depthwise_conv2d(converter, interiors, inputs):
+  x_in = converter.outputs[inputs[0]]
+
+  conv_op = interiors["depthwise"]
+
+  kernel = interiors["depthwise_kernel"]
+  k = _nodef_to_numpy_array(kernel)
+  kernel_init = tf.keras.initializers.Constant(k)
+
+  try:
+    bias = interiors["bias"]
+    b = _nodef_to_numpy_array(bias)
+    bias_init = tf.keras.initializers.Constant(b)
+    use_bias = True
+  except KeyError:
+    use_bias = False
+    bias_init = 'zeros'
+
+  shape = [i.size for i in kernel.attr["value"].tensor.tensor_shape.dim]
+
+  fmt = conv_op.attr["data_format"].s.decode('ascii')
+  fmt = "channels_last" if fmt == "NHWC" else "channels_first"
+
+  strides = int(max(conv_op.attr["strides"].list.i))
```

Comment:
curious, why do we take the max here?

Reply:
@jvmancuso 
because when the stride is 2 then the conv op node looks like this - 
```
attr {
  key: "strides"
  value {
    list {
      i: 1
      i: 2
      i: 2
      i: 1
    }
  }
```

so using max we only extract the 2. 
This is how it was also done in the conv2d method in register.py

Reply:
hm, I think that should be fixed then (for conv2d as well).  we can do that in a separate PR though!